core/unit_test/tstAoSoA.hpp

```diff
@@ -243,42 +251,51 @@ void testRawData()
     // Initialize the data with raw pointer/stride access. Start by looping
     // over the structs. Each struct has a group of contiguous arrays of size
     // array_size for each member.
-    int num_soa = slice_0.numSoA();
-    for ( int s = 0; s < num_soa; ++s )
-    {
-        // Loop over the array in each struct and set the values.
-        int local_array_size = slice_0.arraySize( s );
-        for ( int i = 0; i < local_array_size; ++i )
-        {
-            p0[ s * st0 + i ] = (s + i) * 1.0;
-            p1[ s * st1 + i ] = (s + i) * 2;
-            p3[ s * st3 + i ] = (s + i) * 4;
-            p4[ s * st4 + i ] = (s + i) * 5.0;
-
-            // Member 2 has some extra dimensions so add those to the
-            // indexing. Note this is layout left.
-            for ( int j = 0; j < m2e0; ++j )
-                for ( int k = 0; k < m2e1; ++k )
-                    p2[ s * st2 + j * 16 * m2e1 + k * 16 + i ] =
-                        (s + i + j + k) * 3.0;
-        }
-    }
+    Kokkos::parallel_for(
+        "raw_data_fill",
```

Comment:
I add them out of habit - its actually helped me find errors in the past if Kokkos is compiled in debug mode. Actually it helped me find one this week in this PR because it can be outputted in a Kokkos error message.